yldio/asap-hub
# feat: CRN-944 Split News Controller

- CRN-1049 adding tutorials entity
- discover entitiy accepts tutorials schema as training
- updating the schema
- fix discover fixture
- fix some type issues
- running schema update again
- wip: query discover
- increase coverage
- fix test
- empty commit to trigger pipeline
- CRN-1049 Adding tutorials route
- wip: adding fe copies
- testing tutorials route
- data provider test
- adding tests FE
- add a test for news card
- tutorial chip appears
- some more renaming
- small refactor using tutorial query
- fix lint error
- fix schema
- review: remove unused exports from entity
- review: remove separated queryFetchByIdData
- adding test to check the click in the tutorial card
- review: moving route from /tutorials/id to /discover/tutorial/id
- review: remove unnecessary describe from Tutorial test
- Update apps/crn-frontend/src/discover/tutorials/__tests__/Tutorial.test.tsx
- Update apps/crn-frontend/src/discover/tutorials/__tests__/Tutorial.test.tsx
- review: NewsDetailsPageProps is now a union of NewsResponse and TutorialsResponse with appropriate props
- review: remove waiting for auth0 not to be in the document
- fix broken test
- fix lint error
- Update apps/crn-frontend/src/discover/Routes.tsx
- review: refactor pill label
- review: rename discover/Tutorials to discover/tutorial/TutorialList to avoid confusion
- review: adding route test to the right place
- update the controller
- implement the data provider
- change calendar default import to a named one
